examples/request-certificate/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"log"
+	"net/url"
+
+	"github.com/sigstore/fulcio/pkg/api"
+	"github.com/sigstore/sigstore/pkg/oauthflow"
+	"github.com/sigstore/sigstore/pkg/signature"
+	"github.com/sigstore/sigstore/pkg/signature/options"
+)
+
+var (
+	fulcioUrl    = "https://v1.fulcio.sigstore.dev"
```

Comment:
Thanks so much for doing this! Would it be possible to make this and oidcIssuer into flags. We can do it in a followup if we think it's a good idea also, but just thinking that since (especially in cases of testing, but also) custom deployments of Fulcio, this would be super cool to be able to use it there!

Reply:
I'm of the opinion that these examples should be minimal API/code examples. Having them as usable programs would maybe give the wrong impression?